packages/cli/src/eleventy-plugins/rocketCopy.cjs

```diff
@@ -1,6 +1,6 @@
 const rocketCopy = {
-  configFunction: (eleventyConfig, { _inputDirCwdRelative, filesExtensionsToCopy }) => {
-    eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/**/*.{${filesExtensionsToCopy}}`);
+  configFunction: (eleventyConfig, { _inputDirCwdRelative }) => {
+    eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/**/*`);
```

Comment:
I would assume that copies over EVERY file form docs/**/* which is imho not what we want
should probably be split in 2... probably something like this
```js
eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/_merged_assets/_static/**/*`);
eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/!(*.md|*.html)*`);
```

but I'm not sure so would need to test it 🤔

Reply:
PR in it's current state definitely does copy over the .md files to `_site-dev` but *not* to `_site`

https://app.netlify.com/sites/apollo-elements/deploys/60ddeb4769ef4c0007e2e7b1

if having them in _site-dev is an issue, maybe we can loop back and fix that in a later PR

Reply:
copying all source files will result in a lot of useless files which will get problematic on bigger sites... I just tested it with

```suggestion
    eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/!(*.md|*.html)*`);
    eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/!(_includes|_data|_assets|_merged_data|_merged_includes)*/**/!(*.md|*.html)*`);
    eleventyConfig.addPassthroughCopy(`${_inputDirCwdRelative}/_merged_assets/_static/**/*`);
```

and that seems to work fine.

- for `_assets/_static` it's copying over everything
- for everything else, it's copying over everything except *.html and *.md 

can you verify that it works for your case as well - if so then let's go with that